frontend/pages/hosts/details/cards/About/_styles.scss

```diff
@@ -16,15 +16,20 @@
     .component__tooltip-wrapper__element {
       max-width: 350px;
     }
-
-    &__data {
-      .device-mapping {
-        &__source {
-          color: $ui-fleet-black-75;
-        }
-        &__more {
-          color: $ui-fleet-black-50;
-        }
```

Comment:
These were not being used because `__data` isn't a thing in this component as far as I can see.  Moved them out from under `__data` and now the "More" link is grey as intended.